test/format/test_format.py

```diff
@@ -387,8 +387,8 @@
             "System`OutputForm": "<mrow><mi>Integrate</mi> <mo>[</mo> <mrow><mrow><mi>F</mi> <mo>[</mo> <mi>x</mi> <mo>]</mo></mrow> <mtext>,&nbsp;</mtext> <mrow><mo>{</mo> <mrow><mi>x</mi> <mtext>,&nbsp;</mtext> <mi>a</mi> <mtext>,&nbsp;</mtext> <mrow><mi>g</mi> <mo>[</mo> <mi>b</mi> <mo>]</mo></mrow></mrow> <mo>}</mo></mrow></mrow> <mo>]</mo></mrow>",
         },
         "latex": {
-            "System`StandardForm": "\\int_a^{g\\left[b\\right]} F\\left[x\\right] \uf74cx",
-            "System`TraditionalForm": "\\int_a^{g\\left(b\\right)} F\\left(x\\right) \uf74cx",
+            "System`StandardForm": "\\int_a^{g\\left[b\\right]} F\\left[x\\right] \, dx",
+            "System`TraditionalForm": "\\int_a^{g\\left(b\\right)} F\\left(x\\right) \, dx",
```

Comment:
I believe this should be `\U0001D451x`

https://github.com/Mathics3/mathics-scanner/blob/be4d6de65386e181dd68be3558bee23efdc29b43/mathics_scanner/data/named-characters.yml#L1607-L1615

Reply:
There is a problem with it: the ci still uses mathics-scanner 1.2.4 so if we use that character, we get an error. Once mathics-scanner get released, we should adjust it.

Reply:
Ok, sure